pkgs/build-support/rust/mk-cargo-vendor-deps-script.py

```diff
@@ -0,0 +1,230 @@
+import glob
```

Comment:
Should it has the nix shebang?
```suggestion
#! /usr/bin/env nix-shell
#! nix-shell -i python3 -p python3
```